README.md

```diff
@@ -1,5 +1,13 @@
 # clj-sqs-extended
 
+## Running the tests
+
+Tests can be run directly from the command line using the [clj-cli](https://clojure.org/guides/deps_and_cli) tool.
+
+```
+$> clj -A:test
```

Comment:
what about dependency on docker-compose?

Reply:
OK, I'll have a deeper look to find a convenient way to spinup the necessary localstack environment for local tests, too.